packages/core/test/invoke.test.ts

```diff
@@ -592,17 +539,15 @@ describe('invoke', () => {
       someParentMachine.provide({
         actors: {
           child: () =>
-            fromMachine(
-              createMachine({
-                id: 'child',
-                initial: 'init',
-                states: {
-                  init: {
-                    entry: [sendParent('STOP')]
-                  }
+            createMachine({
+              id: 'child',
+              initial: 'init',
+              states: {
+                init: {
+                  entry: [sendParent('STOP')]
```

Comment:
Note: changes like this will conflict with https://github.com/statelyai/xstate/pull/3568